src/test/java/kitchenpos/application/MenuGroupServiceTest.kt

```diff
@@ -0,0 +1,64 @@
+package kitchenpos.application
+
+import io.kotest.assertions.throwables.shouldThrowExactly
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import kitchenpos.domain.MenuGroup
+import kitchenpos.domain.MenuGroupRepository
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import java.util.*
+
+@ExtendWith(MockKExtension::class)
+internal class MenuGroupServiceTest {
+    @MockK
+    private lateinit var menuGroupRepository: MenuGroupRepository
+
+    @InjectMockKs
+    private lateinit var menuGroupService: MenuGroupService
+
+    @Nested
+    inner class `메뉴 그룹 생성 테스트` {
+        @DisplayName("메뉴 그룹에 이름이 null 또는 길이가 0일 경우, IllegalArgumentException 예외 처리를 한다.")
+        @ParameterizedTest
+        @NullAndEmptySource
+        fun test1(name: String?) {
+            // given
+            val request = MenuGroup().apply {
+                this.id = UUID.randomUUID()
+                this.name = name
+            }
+
+            // when & then
+            shouldThrowExactly<IllegalArgumentException> {
+                menuGroupService.create(request)
+            }
+        }
+
+        @DisplayName("메뉴 그룹이 정상적일경우, 메뉴 그룹이 생성된다.")
+        @Test
+        fun test2() {
+            // given
+            val request = MenuGroup().apply {
+                this.id = UUID.randomUUID()
+                this.name = "테스트 그룹"
+            }
```

Comment:
Fixture의 변수명을 아래의 예시와 같이 한글로 의미있게 정의하면 
처음엔 어색할 수 있으나 직관성을 높여 테스트 케이스에 대한 가독성을 높일 수도 있습니다. 😄 
테스트 데이터도 가능한 실제 데이터와 흡사하게 넣어주는 것이 좋구요.
```suggestion
            val 한마리_메뉴그룹 = MenuGroup().apply {
                this.id = UUID.randomUUID()
                this.name = "한마리 메뉴"
            }
```

Reply:
한글 변수명으로 의도를 나타내면 가독성이 향상되는 것 같습니다!